dodola/services.py

```diff
@@ -176,11 +174,18 @@ def apply_aiqpd(simulation, aiqpd, year, variable, out):
     sim_ds = storage.read(simulation)
     aiqpd_ds = storage.read(aiqpd)
 
-    year = int(year)
+    # zarr dimensions can be garbled, and if dim order
+    # is not lat, lon, dayofyear, quantile, cannot broadcast
+    aiqpd_ds = aiqpd_ds.transpose("lon", "lat", "dayofyear", "quantiles")
```

Comment:
I'd remove this transpose line. Right now (might go back on this later...), transposing feels to me like something that is outside the realm `apply_aiqpd` and its responsibilities. This transpose also risks reading in big data for a very expensive operation. In the case of this PR's integration tests, for example, I would transpose before passing the dataset to any dodola functions.

Reply:
Updated!